__tests__/app.test.js

```diff
@@ -114,3 +114,114 @@ describe("GET /api/reviews/:review_id", () => {
 			});
 	});
 });
+
+describe("GET /api/reviews/:review_id/comments", () => {
+	test("status:200, responds with an array of comment objects", () => {
+		return request(app)
+			.get("/api/reviews/2/comments")
+			.expect(200)
+			.then(({ body }) => {
+				const { comments } = body;
+				expect(comments).toHaveLength(3);
+				comments.forEach((comment) => {
+					expect(comment).toMatchObject({
+						comment_id: expect.any(Number),
+						votes: expect.any(Number),
+						created_at: expect.any(String),
+						author: expect.any(String),
+						body: expect.any(String),
+						review_id: expect.any(Number)
+					});
+				});
+			});
+	});

```

Comment:
still need to add test for if you're getting back an empty array

Reply:
Already done in 7e7b5d65686f164fec9f2a61e8c16ccbd356d38a